tools/ci/tc/tests/test_valid.py

```diff
@@ -310,4 +335,5 @@ def test_schedule_tasks(event_path, is_pr, files_changed, expected):
             with open(data_path(event_path), encoding="utf8") as event_file:
                 event = json.load(event_file)
                 scheduled = decision.decide(event)
+                print(list(scheduled.keys()))
```

Comment:
Did you mean to leave this logging in?

Reply:
Yeah, if you add new jobs it means that the tests will print out the actual list of jobs, which is useful in updating the expectations.